src/gui/properties/peerlistwidget.cpp

```diff
@@ -130,6 +131,11 @@ PeerListWidget::PeerListWidget(PropertiesWidget *parent)
     m_proxyModel->setSourceModel(m_listModel);
     m_proxyModel->setSortCaseSensitivity(Qt::CaseInsensitive);
     setModel(m_proxyModel);
+    // When state array size is 0, this is the first run.
+    // So here we can set the defaults for it.
+    if (Preferences::instance()->getPeerListState().size() == 0) {
```

Comment:
Opening brace in new line (or omit them) and you could re-use `loadSettings()` see: https://github.com/qbittorrent/qBittorrent/blob/3ee0457cfa7ed4ea0850632d675175d304c8d9b8/src/gui/transferlistwidget.cpp#L160